### ISSUE TITLE
Unhandled exception on new install of .Web to Umbraco 13.0.1

### ISSUE THREAD

**Issue by Eaglef90** [AaronSadlerUK/Our.Umbraco.UmbNav#86]
### Which *exact* Umbraco version are you using? For example: 8.7.0 - don't just write v8 or v9

13.0.1

### Bug summary

After installing Our.Umbraco.UmbNav.Web 2.0.8 I get the following error when I start a debug sesttion:

System.Reflection.ReflectionTypeLoadException: 'Could not load all types from "UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null" due to LoaderExceptions, skipping:
. System.TypeLoadException on UmbNav.Core.ValueConnectors.UmbNavValueConnector: Method 'ToArtifact' in type 'UmbNav.Core.ValueConnectors.UmbNavValueConnector' from assembly 'UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.

Method 'ToArtifact' in type 'UmbNav.Core.ValueConnectors.UmbNavValueConnector' from assembly 'UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.'

### Specifics

I setup a new install of 13.0.0 RC5 then upgraded to 13.0.1 then installed UmbNav.Web and now when I do a debug in VS I get this error during startup in program.cs:

```
System.Reflection.ReflectionTypeLoadException: 'Could not load all types from "UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null" due to LoaderExceptions, skipping:
. System.TypeLoadException on UmbNav.Core.ValueConnectors.UmbNavValueConnector: Method 'ToArtifact' in type 'UmbNav.Core.ValueConnectors.UmbNavValueConnector' from assembly 'UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.

Method 'ToArtifact' in type 'UmbNav.Core.ValueConnectors.UmbNavValueConnector' from assembly 'UmbNav.Core, Version=2.0.3.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.'
```

If I uninstall the package the error goes away

### Steps to reproduce

Setup Umb13
Install UmbNav.Web 2.0.8
Run a debug in Visual Studio

### Expected result / actual result

Expected Result:
No error is thrown.

Actual Result:
A breaking error is thrown.

**Comment by AaronSadlerUK** [comment]
This hasn't been updated for V13 yet, see issue #84

**Comment by Eaglef90** [comment]
My apologies. I remember seeing the title of that issue when I was searching for an answer for my core vs web question and I guess my head remembered it as completed.

**Comment by AaronSadlerUK** [comment]
Fixed in V3.0.0 which is on NuGet now
